guides/common/modules/proc_browsing-repository-content-using-an-organization-debug-certificate.adoc

```diff
@@ -36,11 +36,11 @@ $ openssl pkcs12 -keypbe PBE-SHA1-3DES -certpbe PBE-SHA1-3DES -export -in cert.p
 . In the Firefox browser, navigate to *Edit* > *Preferences* > *Advanced Tab*.
 . Select *View Certificates*, and click the *Your Certificates* tab.
 . Click *Import* and select the `.pfx` file to load.
-. In the address bar, enter a URL in the following format to browse for repositories:
+. In the address bar, enter the *Published At* URL of a repository to browse it:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-http://_{foreman-example-com}_/pulp/repos/_organization_label_
+https://_{foreman-example-com}_/pulp/content/organization_label/Library/content/dist/rhel/server/7/7Server/x86_64/os/_
```

Comment:
Should this be
```suggestion
https://_{foreman-example-com}_/pulp/content/_organization_label_/Library_/content/dist/rhel/server/7/7Server/x86_64/os/
```

so that `_organization_label_` is still italics? also, using 7Server explicitly seems like an odd choice to me?

Reply:
> Yeah, the "problem" is that in pulp2, you could go to `/pulp/repos/<org>` and see what you have -- but on pulp3 you can't do that, and get only a 404. What you can do is to go to `/pulp/content/` and see _everything_, which I think is better than pointing to an explicit repo (even if existing).
> 
> Honestly, this feels like a missing feature (or config error?) in pulp3 to me, but we're not going to fix that trivially today, so probably should document what's working today.

I agree with this as that is part of the power of browsing with the debug certificate. Including one directed example works in addition to this; noting you can visit the root `pulp/content` to see all available items to look at is worth a note.

Reply:
+1 I have commented in the BZ now and I believe, It's good to cover the `/pulp/content/` part there in Point 5 i.e.

~~~
In the address bar, enter the following URL to browse the accessible paths for all the repositories and further check their contents:

https://{foreman-example-com}/pulp/content/
~~~

@mjivraja FYI

Reply:
So, by going through the comments, I assume there is no modification required for URLs. The only thing is keeping `_organization_label_` in italics.

@evgeni @ehelms @sayan3296 Please confirm.

Reply:
@mjivraja I hope after our small discussion on g-chat, You are clear about the newest suggestion made above.

Reply:
Yes. Thank you for clarifying 👍

Reply:
The Builds were successful locally for all. This is a new error for me. :worried: